themes/edx.org/lms/templates/course_modes/choose.html

```diff
@@ -102,7 +108,14 @@ <h4 class="title">Pursue a Verified Certificate</h4>
                                     <li class="action action-select">
                                       <input type="hidden" name="contribution" value="${min_price}" />
                                       <!-- The class verified_mode should be added to this selector if the experiment is implemented-->
-                                      <input type="submit" name="verified_mode" value="Upgrade to a Certificate ($${min_price} USD)" />
+                                      <div class="upgradev1">
+                                        <input type="submit" name="verified_mode" value="Upgrade to a Certificate ($${min_price} USD)" />
+                                      </div>
+                                      <div class="upgradev2 hidden">
+                                          <button type="submit" name="verified_mode">Upgrade to a Certificate (<del>$${min_price} USD</del>)</button>
+                                          <br>
+                                          <span class="savings-message">Save 5% if you upgrade now! ($${int(min_price * .95)} USD)</span>
```

Comment:
What about marking this and other new strings for i18n? We've been doing it with other changes.